eth-contracts/migrations/4_governance_migration.js

```diff
@@ -16,19 +15,20 @@ const VotingQuorum = 1
 
 module.exports = (deployer, network, accounts) => {
   deployer.then(async () => {
-    const protocolOwner = accounts[0]
-    // TODO move to contractConfig
-    const [proxyAdminAddress, proxyDeployerAddress] = [accounts[10], accounts[11]]
+    const config = contractConfig[network]
+    const proxyAdminAddress = config.proxyAdminAddress || accounts[10]
+    const proxyDeployerAddress = config.proxyDeployerAddress || accounts[11]
     
-    const registry = await Registry.deployed()
-
+    const registryAddress = process.env.registryAddress
```

Comment:
should throw if this value is not available i think, since now we rely on this - like if(!registryAddress) throw

Reply:
same for any env based address access throughout

Reply:
let's do separately